test/torchaudio_unittest/common_utils/case_utils.py

```diff
@@ -112,22 +113,8 @@ class TorchaudioTestCase(TestBaseMixin, PytorchTestCase):
     pass
 
 
-_IS_FFMPEG_AVAILABLE = None
-
-
 def is_ffmpeg_available():
-    if _eval_env("TORCHAUDIO_TEST_IN_FBCODE", default=False):
-        return True
-
-    global _IS_FFMPEG_AVAILABLE
-    if _IS_FFMPEG_AVAILABLE is None:
-        try:
-            from torchaudio.io import StreamReader  # noqa: F401
-
-            _IS_FFMPEG_AVAILABLE = True
-        except Exception:
-            _IS_FFMPEG_AVAILABLE = False
-    return _IS_FFMPEG_AVAILABLE
+    return torchaudio._extension._FFMPEG_INITIALIZED
```

Comment:
does this change relate to log level configurability?

Reply:
not technically. but as I write the test, I realized that the logic is duplicated in library and test util.